src/main/java/org/mindrot/jbcrypt/BCrypt.java

```diff
@@ -0,0 +1,777 @@
+// Copyright (c) 2006 Damien Miller <djm@mindrot.org>
```

Comment:
I don't like this too much. I'm sure this can be found in some library in Maven central, Bouncy Castle seems to have support for BCrypt for example ?